config/configuration.json

```diff
@@ -436,7 +436,16 @@
       "tags": "building"
     }
   },
-  "LineCrossingWaterBodyCheck": {},
+  "LineCrossingWaterBodyCheck": {
+    "enabled": true,
```

Comment:
nit: All the checks are enabled by default in this config.

Reply:
We also need the instructions and description for this check.

Reply:
Sorry @dcastrowa , jumped the gun on the merge. Do one of you two mind adding the MR challenge details?

Reply:
Yup, this is going up with a PR very soon.

Reply:
added these updates here: https://github.com/osmlab/atlas-checks/pull/266